install/kubernetes/helm/istio/templates/tests/test-grafana-connection.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if .Values.global.enabledTest }}
+{{- if .Values.grafana.enabled }}
+apiVersion: v1
+kind: Pod
+metadata:
+  name: {{ template "grafana.fullname" . }}-test
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: grafana-test
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    istio: grafana
+  annotations:
+    scheduler.alpha.kubernetes.io/critical-pod: ""
+    sidecar.istio.io/inject: "false"
+    helm.sh/hook: test-success
+spec:
+{{- if .Values.global.priorityClassName }}
+  priorityClassName: "{{ .Values.global.priorityClassName }}"
+{{- end }}
+  containers:
+    - name: "{{ template "grafana.fullname" . }}-test"
+      image: {{ .Values.global.hub }}/{{ .Values.global.proxy.image }}:{{ .Values.global.tag }}
+      imagePullPolicy: "{{ .Values.global.imagePullPolicy }}"
+      command: ['curl']
+      args: ['http://grafana:{{ .Values.grafana.service.externalPort }}']
```

Comment:
I had thought but am not certain externalPort was in the process of deprecation?

Reply:
I think `internalPort` for grafana` is  deprecation for this PR: https://github.com/istio/istio/pull/6090/files
but `externalPort ` should be working. 
Am I missing something?

Reply:
@morvencao I don't recall specifics - at some point someone suggested removing externalPort.  But it could have been internalPort.  I don't know if that work merged.  If this PR passes the gate, wfm.

/lgtm
/approve